hasura/hasura-metadata/tables.yaml

```diff
@@ -48,6 +48,34 @@
           table:
             name: deposits
             schema: public
+    - name: group_members
+      using:
+        foreign_key_constraint_on:
+          column: account_id
+          table:
+            name: group_members
+            schema: public
+    - name: group_policies
+      using:
+        foreign_key_constraint_on:
+          column: admin
+          table:
+            name: group_policies
+            schema: public
+    - name: group_votes
+      using:
+        foreign_key_constraint_on:
+          column: voter_id
```

Comment:
group_proposal_id?

Reply:
I think you are doing right, haha, you can treat this like a note where I don't understand